Products/CMFPlone/skins/plone_scripts/getObjSize.py

```diff
@@ -8,6 +8,7 @@
 ##title=
 
 from Products.CMFPlone.utils import base_hasattr
+import six
```

Comment:
Oh, crap: `Unauthorized: import of 'six' is unauthorized`

Reply:
Crap indeed. It *might* be okay to call `allow_module` on six, but that would need more investigation on what exactly is in six, and if that is safe to allow. I guess it is better to do this by hand then...

Reply:
Good news, I confirmed `isinstance` is allowed

Reply:
Problem is you have to check for long because of large file sizes but cannot know in advance if it will be available because there's no access to sys.version_info or builtins to check if RestrictedPython is running in Python 3. @loechel, is it possible?

Otoh we could try to subset `six` (RestrictedSix?) before allowing it: https://github.com/benjaminp/six/blob/master/six.py#L235-L303

Reply:
I would say keep it simple like this:
```
try:
    integer_types = (int, long)
except NameError:
    # Python 3
    integer_types = (int, )
```

Reply:
Or replace that with a browser view/utility :-) aren't we supposed to get rid of all those skins? 😆